vrtUtil/vrtUtil.js

```diff
@@ -0,0 +1,77 @@
+const path = require('path');
+const fs = require('fs');
+const ImageDiff = require('./ImageDiff');
+
+const VISUAL_REGRESSION_THRESHOLD = 0.01;
```

Comment:
nit: if this a percentage? if so, we can name it a bit more clearly

Reply:
will add a comment